test_ert_integration_errors fails in Komodo bleeding-py3.8

**Describe the bug**
In bleeding komodo Python 3.8 the following test fails: 

```
_________________________ test_ert_integration_errors __________________________

norne_mocked_ensembleset = None

    @pytest.mark.skipif(sys.version_info < (3, 7), reason="Py 3.7 for capture_output")
    @pytest.mark.integration
    def test_ert_integration_errors(norne_mocked_ensembleset):
        """Test CSV_EXPORT2 when runpathfile points to non-existing realizations
    
        This test proves that CSV_EXPORT2 happily skips non-existing
        realizations, but emits a warning that there is no STATUS file.
        """
        with open("FOO.DATA", "w", encoding="utf-8") as file_h:
            file_h.write("--Empty")
    
        # Append a not-existing realizations to the runpathfile:
        with open("runpathfile", "a", encoding="utf-8") as file_h:
            file_h.write("002 realization-2/iter-0 NORNE_1 000")
    
        with open("wf_csvexport", "w", encoding="utf-8") as file_h:
            file_h.write("CSV_EXPORT2 runpathfile data.csv monthly FOPT\n")
    
        ert_config = [
            "ECLBASE FOO.DATA",
            "QUEUE_SYSTEM LOCAL",
            "NUM_REALIZATIONS 2",
            "LOAD_WORKFLOW wf_csvexport",
            "HOOK_WORKFLOW wf_csvexport PRE_SIMULATION",
        ]
    
        ert_config_fname = "test.ert"
        with open(ert_config_fname, "w", encoding="utf-8") as file_h:
            file_h.write("\n".join(ert_config))
    
        ertoutput = subprocess.run(
            ["ert", "test_run", ert_config_fname], check=True, capture_output=True
        )
    
>       assert "fmu.ensemble.realization - WARNING - No STATUS file" in str(
            ertoutput.stdout
        )
E       assert 'fmu.ensemble.realization - WARNING - No STATUS file' in "b'monthly csv-export written to data.csv\\n    --> Running single realisation test ...\\n\\n\\r    |                 .../1\\n    Failed            0/1\\n    Finished          1/1\\n    Unknown           0/1\\n\\nSimulations completed.\\n'"
E        +  where "b'monthly csv-export written to data.csv\\n    --> Running single realisation test ...\\n\\n\\r    |                 .../1\\n    Failed            0/1\\n    Finished          1/1\\n    Unknown           0/1\\n\\nSimulations completed.\\n'" = str(b'monthly csv-export written to data.csv\n    --> Running single realisation test ...\n\n\r    |                      ...      0/1\n    Failed            0/1\n    Finished          1/1\n    Unknown           0/1\n\nSimulations completed.\n')
E        +    where b'monthly csv-export written to data.csv\n    --> Running single realisation test ...\n\n\r    |                      ...      0/1\n    Failed            0/1\n    Finished          1/1\n    Unknown           0/1\n\nSimulations completed.\n' = CompletedProcess(args=['ert', 'test_run', 'test.ert'], returncode=0, stdout=b'monthly csv-export written to data.csv\n...thon 3.8, and scheduled for removal in Python 3.10.\n  timeout_queue = asyncio.Queue(loop=asyncio.get_event_loop())\n').stdout

/tmp/jenkins-komodo-f_scout_ci/workspace/unit-tests/semeio/test-root/tests/jobs/csv_export2/test_integration.py:268: AssertionError
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run semeio unit tests against bleeding 3.8

**Expected behavior**
No failure

--- a/tests/jobs/csv_export2/test_integration.py
+++ b/tests/jobs/csv_export2/test_integration.py
@@ -261,14 +261,14 @@ def test_ert_integration_errors(norne_mocked_ensembleset):
     with open(ert_config_fname, "w", encoding="utf-8") as file_h:
         file_h.write("\n".join(ert_config))
 
-    ertoutput = subprocess.run(
+    subprocess.run(
         ["ert", "test_run", ert_config_fname], check=True, capture_output=True
     )
 
-    assert "fmu.ensemble.realization - WARNING - No STATUS file" in str(
-        ertoutput.stdout
-    )
-    assert "realization-2/iter-0" in str(ertoutput.stdout)
+    with open("ert-log.txt") as fin:
+        ertlog = fin.read()
+    assert "fmu.ensemble.realization - WARNING - No STATUS file" in ertlog
+    assert "realization-2/iter-0" in ertlog
 
     assert os.path.exists("data.csv")
     assert pd.read_csv("data.csv").shape == (16, 5)

